README.rst

```diff
@@ -107,8 +107,22 @@ Since changing the editor is a pretty big deal, there are a number of feature fl
 
 Building a CKEditor Release
 ===========================
+First, you need to build a CKEditor release version.
+Clone this repo in a separate folder for release building.
+For instance, do this ``git clone git@github.com:PolicyStat/ckeditor-dev.git ckeditor-release``
 
-First, remove
+:warning: Strictly do not do ``npm install`` in the release folder.
+``npm install`` would install development dependencies and lock them in ``package-lock.json``.
+This leads to false alarms from ``github.com``
```

Comment:
Are these false alarms? Or just things we are not addressing? Can you please also mention that it's security issues that are being raised as alarms? And maybe also include why we are circumventing the security checks?

Reply:
Yes they are.

1. These alarms produced by `package-lock.json`
![image](https://user-images.githubusercontent.com/669017/70465990-381bf780-1ad3-11ea-8b38-936f8062dce9.png)

2. `package-lock` contains dev dependencies only https://github.com/PolicyStat/ckeditor-dev/blob/pstat/package.json#L5-L26

3. CKEditor does their release to npm from the separate repo https://github.com/ckeditor/ckeditor4-releases/blob/master/package.json and there are no dependencies. In other words CKEditor is inedependant package.

Reply:
Can we please update the docs to reflect everything in this comment thread? I'd rather not need to know to look here to answer these questions.

Reply:
Done ✔️